Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -151,6 +151,47 @@ TEST_F(SerializerTest, EncodesIntegersModelToBytes) {
   }
 }
 
+TEST_F(SerializerTest, EncodesStringModelToBytes) {
+  struct TestCase {
+    std::string value;
+    std::vector<uint8_t> bytes;
+  };
+
+  std::vector<TestCase> cases{
+      // TEXT_FORMAT_PROTO: 'string_value: ""'
+      {"", {0x8a, 0x01, 0x00}},
+      // TEXT_FORMAT_PROTO: 'string_value: "a"'
+      {"a", {0x8a, 0x01, 0x01, 0x61}},
+      // TEXT_FORMAT_PROTO: 'string_value: "abc def"'
+      {"abc def", {0x8a, 0x01, 0x07, 0x61, 0x62, 0x63, 0x20, 0x64, 0x65, 0x66}},
+      // TEXT_FORMAT_PROTO: 'string_value: "æ"'
+      {"æ", {0x8a, 0x01, 0x02, 0xc3, 0xa6}},
```

Comment:
absolutely optional: Is it possible to use escape-sequence for the unicode character here instead of use that character directly? Assuming any modern editor/source-control-system supports unicode, there is no problem; so it is optional.

Reply:
We could; but OTOH, we've already got some test cases that use escape characters below. (More context: These test cases are taken from the other test suites, so my preference is to leave them as close as possible to the original.)

We could make an argument that this is redundant... but it's probably more trouble than it's worth to remove it across all platforms.